tools/tuf/test/create_repository.sh

```diff
@@ -61,13 +61,9 @@ for system in $SYSTEMS; do
     rm $osqueryd_path
 
     goose_value="$system"
-    goarch_value="" # leave it empty to use the default for the system
+    goarch_value=${GOARCH:-amd64}
```

Comment:
hey, sorry to keep going adding review comments 🙏 

I think this is a step backwards because it now forces people on arm64 to set the goarch again? I was thinking of the env var as a mechanism to force the arch if you need to.